src/node_v8_platform-inl.h

```diff
@@ -128,19 +128,18 @@ struct V8Platform {
 
   inline void StartTracingAgent() {
     constexpr auto convert_to_set =
-        [](std::vector<std::string_view> categories) -> std::set<std::string> {
+        [](auto& categories) -> std::set<std::string> {
       std::set<std::string> out;
       for (const auto& s : categories) {
-        out.emplace(s);
+        out.emplace(std::string(s.data(), s.size()));
```

Comment:
```suggestion
        out.emplace(std::string(s));
```

Reply:
Unfortunately, this doesn't work. std::string doesn't have a constructor for whatever the type `std::views::split` returns

Reply:
Me == sad.

Reply:
It is a subrange... which is kind of like a string_view, but more general (not sure for strings).